lmp/jobserv.yml

```diff
@@ -126,6 +126,13 @@ triggers:
       OTA_LITE_TAG: 'scarthgap'
       AKLITE_TAG: promoted-scarthgap
     runs:
+      - name: lmp-sdk
+        host-tag: amd64
+        container: foundries/dind-ci:19.03.9_b38f166
```

Comment:
Why are we using this old tag for lmp-sdk?

Reply:
we only run this lmp-sdk in the build-kirkstone and I copy it also to build-scarthgap and build-main. the tag is the same

Reply:
I have dropped the job to build the container from the build-kirkstone and build-main and keep it only on the build-scarthgap.

Reply:
I will add in another [PR381](https://github.com/foundriesio/ci-scripts/pull/381) the change to build it on build-main but for this I need @doanac help because we need to tag the container with a different name like _next_ or something to only be used on the main branch.